src/server/config/schema.js

```diff
@@ -130,6 +132,21 @@ module.exports = () => Joi.object({
 
   status: Joi.object({
     allowAnonymous: Joi.boolean().default(false)
+  }).default(),
+
+  tilemap: Joi.object({
+    url: Joi.string().default(`https://tiles.elastic.co/v1/default/{z}/{x}/{y}.png?my_app_name=kibana&my_app_version=${pkg.version}&elastic_tile_service_tos=agree`),
+    options: Joi.object({
+      attribution: Joi.string().default('© [Elastic Tile Service](https://www.elastic.co/elastic-tile-service)'),
+      minZoom: Joi.number().min(1, 'Must not be less than 1').default(1),
+      maxZoom: Joi.number().default(7),
+      tileSize: Joi.number().optional(),
+      subdomains: Joi.array().items(Joi.string()).single().optional(),
+      errorTileUrl: Joi.string().uri().optional(),
+      tms: Joi.boolean().optional(),
+      reuseTiles: Joi.boolean().optional(),
+      bounds: Joi.array().items(Joi.array().items(Joi.number()).min(2).required()).min(2).optional()
+    }).default()
```

Comment:
That's on me, but I preferred the readability because I was mixing it with ones that were defined with `default(...)` and most did not.